### PR TITLE
Using new ingestion endpoint

### PR DESCRIPTION
**Pull request by imnutz**

### CODE REVIEW

File: lib/plugins/track.js [hunk 1]
```diff
@@ -64,7 +64,9 @@ function configureValues (track) {
           : document.location.href.split('#')[0]
       },
       td_user_agent: function () {
-        return window.navigator.userAgent
+        var userAgent = window.navigator.userAgent
+
+        return [userAgent, 'JSSDK/' + version].join(';')
```
Comment (tung-vu-td) [inline lib/plugins/track.js:69]: Nitpicking:
Either declare both `userAgent` and `sdkUserAgent`, or inline `window.navigator.userAgent` to this line.

**Review by tung-vu-td — APPROVED** [review]
LGTM

File: STREAMING_INGESTION.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+- AWS Asia Pacific (Tokyo):
+  - **ap01.records.in.treasuredata.com**
+
+- AWS East
+  - **us01.records.in.treasuredata.com**
+
+- AWS EU
+  - **eu01.records.in.treasuredata.com**
+
+The `path` configuration should be always `/`
```
Comment (cyberdelia) [inline STREAMING_INGESTION.md:21]: Can we just remove the option then?

Comment (imnutz) [reply STREAMING_INGESTION.md]: The property should be `pathname`, as it is mentioned in the configuration section of the documentation. We want to explicitly set the `pathname` to `/`

**Review by ylzen — APPROVED** [review]
Reviewed for Data Exfiltrations.

**Pull request merged**